Orange/widgets/cpewidget.py

```diff
@@ -59,14 +67,20 @@ def reset_default_properties(self, names=None):
 
         for name in (names or self.get_property_names()):
             setattr(self.node, name, getattr(node, name))
+            value = getattr(self.node, name)
             # Synchronize property changes back to the GUI.
-            super().__setattr__(name, getattr(self.node, name))
+            if (isinstance(self.node.port(name), ListPort)
```

Comment:
there are 3 different places in the code where the same kind of (pretty messy) check happens...
basically whenever you do super().__setattr__(name, value) and value would be None, you want to rewrite it by NoneUPValue[0]. How about you make this piece of logic a method? (e.g. assign_value_to_widget(self, value)) If you're supporting every kind of port anyway, then you no longer need the isinstance check at all, actually.